### PR TITLE
Bump ks-devops chart from 0.1.5 to 0.1.8

### PR DESCRIPTION
**Pull request by LinuxSuRen**
/cc @kubesphere/sig-devops

### CODE REVIEW

File: roles/download/defaults/main.yml [hunk 1]
```diff
@@ -101,21 +101,21 @@ ks_devops_tools_tag: "{{ ks_version }}"
...
-builder_registry: "{{ base_repo }}{{ namespace_override | default('kubesphere') }}"
-builder_base_tag: v3.1.0
-builder_nodejs_tag: v3.1.0
-builder_maven_tag: v3.1.0
-builder_go_tag: v3.1.0
+builder_registry: "{{ base_repo }}{{ namespace_override | default('kubespheredev') }}"
+builder_base_tag: v3.2.0
+builder_nodejs_tag: v3.2.0
+builder_maven_tag: v3.2.0
+builder_go_tag: v3.2.0
```
Comment (JohnNiang) [inline roles/download/defaults/main.yml:112]: First, we have to add a new image `builder-go:v3.2.0-1.16` into `images`.
image
Then, add `builder_go16_tag: v3.2.0-1.16` for `images.builder_go16` use:
image

Comment (LinuxSuRen) [reply roles/download/defaults/main.yml:112]: Thanks for your reminder.

File: roles/download/defaults/main.yml [hunk 2]
```diff
@@ -101,21 +101,22 @@ ks_devops_tools_tag: "{{ ks_version }}"
...
 
 #s2i
-s2i_registry: "{{ base_repo }}{{ namespace_override | default('kubesphere') }}"
-s2ioperator_tag: "v3.1.0"
-s2irun_tag: "v2.1.1"
-s2itemplates_tag: "v2.1.0"
+s2i_registry: "{{ base_repo }}{{ namespace_override | default('kubespheredev') }}"
+s2ioperator_tag: "v3.2.0"
+s2irun_tag: "v3.2.0"
+s2itemplates_tag: "v3.2.0"
```
Comment (JohnNiang) [inline roles/download/defaults/main.yml:119]: Please note that not all s2i templates are released to `v3.2.0`. Please see 
- https://github.com/kubesphere/s2i-java-container/releases.
- https://github.com/search?q=org%3Akubesphere+s2i

File: roles/download/defaults/main.yml [hunk 3]
```diff
@@ -408,6 +409,12 @@ images:
     sha256: "{{ builder_go_checksum|default(None) }}"
     group: "kubesphere-devops-images"
 
+  builder_go16:
+    repo: "{{ builder_registry }}/builder-go"
+    tag: "{{ builder_go16_tag }}"
+    sha256: "{{ builder_go_checksum|default(None) }}"
```
Comment (JohnNiang) [inline roles/download/defaults/main.yml:415]: IMO, here is suppose to be a checksum method different from `builder_go:v3.2.0`. I'd prefer to use `builder_go16_checksum`.

File: roles/download/defaults/main.yml (hunk 2 above)
Comment (JohnNiang) [reply roles/download/defaults/main.yml:119]: There are two potential solutions here:

1. Set tag separately for s2i templates' images.
2. Release unreleased s2i templates.

Comment (JohnNiang) [reply roles/download/defaults/main.yml:119]: Please refer to https://github.com/kubesphere-sigs/ks-devops-helm-chart/pull/34/files

Comment (LinuxSuRen) [reply roles/download/defaults/main.yml:119]: See https://github.com/kubesphere-sigs/ks-devops-helm-chart/pull/37

File: roles/download/defaults/main.yml [hunk 4]
```diff
@@ -101,21 +101,22 @@ ks_devops_tools_tag: "{{ ks_version }}"
 
 #jenkins:
 jenkins_repo: "{{ base_repo }}{{ namespace_override | default('kubespheredev') }}/ks-jenkins"
-jenkins_tag: 3.2.0-rc0-2.249.1
+jenkins_tag: 3.2.0-2.249.1
```
Comment (pixiake) [inline roles/download/defaults/main.yml:104]: Is `v` (v3.2.0-2.249.1) needed here?

**Comment by JohnNiang** [comment]
/retitle Bump ks-devops chart from 0.1.5 to 0.1.7

File: roles/download/defaults/main.yml (hunk 4 above)
Comment (LinuxSuRen) [reply roles/download/defaults/main.yml]: Yes

**Pull request merged**